src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -43,6 +43,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <UseTargetPlatformAsNuGetTargetMoniker Condition="'$(UseTargetPlatformAsNuGetTargetMoniker)' == '' AND '$(TargetFrameworkMoniker)' == '.NETCore,Version=v5.0'">true</UseTargetPlatformAsNuGetTargetMoniker>
     <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == '' AND '$(UseTargetPlatformAsNuGetTargetMoniker)' == 'true'">$(TargetPlatformIdentifier),Version=v$([System.Version]::Parse('$(TargetPlatformMinVersion)').ToString(3))</NuGetTargetMoniker>
     <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == '' AND '$(UseTargetPlatformAsNuGetTargetMoniker)' != 'true'">$(TargetFrameworkMoniker)</NuGetTargetMoniker>
+
+    <EmitAssetsLogMessages Condition="'$(EmitAssetsLogMessages)' == ''">false</EmitAssetsLogMessages>
```

Comment:
This should be true by default. I set it to false because it is failing many tests until we get an updated CLI (with updated NuGet) into the sdk. Also NuGet wants to consume this property to determine whether to emit the diagnostics to the error list/command line themselves (i.e. only if it is false or not defined). So we may need to set it to true in a different target file as well, if this one is not appropriate.